huawei-wmi.c

```diff
@@ -140,10 +139,10 @@ static const struct key_entry huawei_wmi_keymap[] = {
 	// Refresh rate (Fn+R)
 	{ KE_KEY,     0x2a7,              { KEY_REFRESH_RATE_TOGGLE } },
 	// Keyboard backlight (space bar, toggles in that order)
-	{ KE_KEY,     KBDLIGHT_KEY_OFF,   { KEY_KBDILLUMDOWN } },
+	{ KE_KEY,     KBDLIGHT_KEY_OFF,   { KEY_KBDILLUMTOGGLE } },
 	{ KE_KEY,     KBDLIGHT_KEY_AUTO,  { KEY_KBDILLUMTOGGLE } },
-	{ KE_KEY,     KBDLIGHT_KEY_LOW,   { KEY_KBDILLUMDOWN } },
-	{ KE_KEY,     KBDLIGHT_KEY_HIGH,  { KEY_KBDILLUMUP } },
+	{ KE_IGNORE,  KBDLIGHT_KEY_LOW,   { KEY_KBDILLUMDOWN } },
+	{ KE_IGNORE,  KBDLIGHT_KEY_HIGH,  { KEY_KBDILLUMUP } },
```

Comment:
These might break on devices that use the driver's quirks and/or `handle_kbdlight` and `kbdlight_auto`

Reply:
To my understanding they won't, as those are just the new ones added by me in #93.